iset.mm

```diff
@@ -163858,6 +163858,23 @@ According to Wikipedia ("Integer", 25-May-2019,
       SUNUOUPVGWAVHELUQQVPCRRURUTVQSTRVAUTVBARBVCVDVEVF $.
   $}
 
+  ${
+    $d x N $.
+    znzrhfo.y $e |- Y = ( Z/nZ ` N ) $.
+    znzrhfo.b $e |- B = ( Base ` Y ) $.
+    znzrhfo.2 $e |- L = ( ZRHom ` Y ) $.
+    $( The ` ZZ ` ring homomorphism is a surjection onto ` ZZ / n ZZ ` .
```

Comment:
should ` ZZ / n ZZ ` be ` Z/nZ ` ?

Reply:
Sure. They look very similar once rendered to HTML so I don't see any reason to keep ` ZZ / n ZZ `